cmd/get_plan_info.go

```diff
@@ -33,7 +33,7 @@ func init() {
 			logger := lager.NewLogger("get_plan_info_cmd")
 			db := db_service.SetupDb(logger)
 
-			var pds []*PlanDetails
+			var pds []*models.PlanDetailsV1
```

Comment:
The reason we use a V1 here is the original struct was deleted as part of the 4.0 work Colleen did and it only hangs around for helping people migrate from 3.x -> 4.0